src/microbe_stage/Microbe.cs

```diff
@@ -1382,6 +1382,13 @@ internal void OnColonyMemberRemoved(Microbe microbe)
             RevertNodeParent();
             ai?.ResetAI();
 
+            // Colony reset at the end, to allow for previous operations on it.
+            Colony = null;
```

Comment:
Why was this added here, as there's already:
https://github.com/Revolutionary-Games/Thrive/blob/cf56b89a03d8dbf5067a260d14e319170fc6d1b4/src/microbe_stage/MicrobeColony.cs#L61
?
I think a better design would be to reset allOrganellesDivided to false when joining a colony (and then make sure that after unbind the reproduction progress checks will properly enable the editor if needed).

Reply:
Hm, that's indeed a good point. But I added this because I realised that, at this point, the colony was not set to null (commenting my line restores the issue).

After investigating, this line in facts sets the microbe Colony to null, but after Microbe-related methods (those which allow to reproduce). I tried moving it, but it disables full unbinding of colony, because, well, the colony to unbind is null. It would make sense, however to delete the `microbe.Colony = null;` line in MicrobeColony.

Reply:
Did you try using the allOrganellesDivided variable approach? I think it might remove the need to fiddle with where the Colony variable is set to null.

Reply:
It would only work for disabling editor, not reenabling it : to activate the editor button from the HUD, I have to use MicrobeHUD.ShowReproductionDialog(); through MicrobeStage.OnPlayerReproductionStatusChanged, which checks player.Colony == null. 

So I need to nullify the Colony beforehand, or change/circumvene other files safety check (which is not advisable in general).

Reply:
Then setting allOrganellesDivided to false here, might work? I think the logic would go like this:
- microbe runs update
- microbe leaves a colony
- microbe runs update (reproduction progress gets re-checked here, and Colony is now null)

Reply:
I wanted to rely on such logic at first but did not consider allOrganellesDivided shortcuts. It may work, in fact. I'll take a look at it after reworking AI code

Reply:
You were fully right on this it seems to work perfectly well, including when saving & loading. It seems much nicer as well. Thanks!